### PR TITLE
Phene

### PR DESCRIPTION
**Pull request by meghan06**

### CODE REVIEW

**Comment by joebobbio** [comment]
LGTM!!!

**Comment by WeirdTreeThing** [comment]
certified phene moment

File: src/docs/installing/post-install.md [hunk 1]
```diff
@@ -68,3 +68,8 @@ If you experience issues in applications such as Parsec, or encounter disruptive
 * [Remove Verbose](https://dortania.github.io/OpenCore-Post-Install/cosmetic/verbose.html#macos-decluttering)
 * [OpenCore GUI Setup](https://dortania.github.io/OpenCore-Post-Install/cosmetic/gui.html#setting-up-opencore-s-gui)
 * Disable `DevirtualiseMmio` if you're on CometLake (10th gen)
+
+
+
```
Comment (ethanaobrien) [inline src/docs/installing/post-install.md:73]: Why so many newlines

**Comment by ninelore** [comment]
what is this?

**Comment by ethanaobrien** [comment]
https://discord.com/channels/232609321274310656/1121994891950559283/1201285863397662805

**Pull request closed without merging**